src/v/datalake/translation/scheduling.cc

```diff
@@ -22,11 +23,14 @@ class default_reservations_tracker : public reservations_tracker {
     explicit default_reservations_tracker(
       size_t total_memory,
       size_t block_size,
-      scheduling_notifications& notifier)
+      scheduling_notifications& notifier,
+      disk_manager& disk_manager)
       : _total_memory((total_memory / block_size) * block_size)
       , _available_memory{_total_memory, "dl/translation/memory"}
       , _reservation_block_size(block_size)
-      , _notifier(notifier) {
+      , _notifier(notifier)
+      , _available_disk{0, "dl/translation/disk"}
```

Comment:
nit: i know it's redundant/obvious once we contextualize ourselves that we're in the reservation tracker, but can we call this `_shard_available_disk`? When grokking this for the first time, or in the heat of an incident, i think we'll be jumping back and forth between here and the datalake/disk manager and it'd really help keep things straight if it were explicit that one is shard local and the other is global

Reply:
fixed